video_worker/api_communicate.py

```diff
@@ -258,14 +249,5 @@ def send_val_data(self):
             )
 
             if r2.status_code > 299:
-                ErrorObject().print_error(
-                    message='VAL POST/PUT Fail'
-                )
+                logger.error('VAL POST/PUT Fail')
                 return None
-
-
-def main():
```

Comment:
@yro @muzaffaryousaf I don't think we are doing anything here, so better to remove this, right?